learning_paths/api/v1/urls.py

```diff
@@ -19,27 +19,27 @@
 
 urlpatterns = router.urls + [
     path(
-        "learning-paths/<uuid:learning_path_uuid>/progress/",
+        "<uuid:learning_path_uuid>/progress/",
         LearningPathUserProgressView.as_view(),
         name="learning-path-progress",
     ),
     path(
-        "learning-paths/<uuid:learning_path_uuid>/grade/",
+        "<uuid:learning_path_uuid>/grade/",
         LearningPathUserGradeView.as_view(),
         name="learning-path-grade",
     ),
     path(
-        "learning-paths/<uuid:learning_path_id>/enrollments/",
+        "<uuid:learning_path_id>/enrollments/",
         LearningPathEnrollmentView.as_view(),
         name="learning-path-enrollments",
     ),
     path(
-        "learning-paths/enrollments/",
+        "enrollments/",
         ListEnrollmentsView.as_view(),
         name="list-enrollments",
     ),
     path(
-        "learning-paths/enrollments/bulk-enroll/",
+        "enrollments/bulk-enroll/",
```

Comment:
@tecoholic, I removed this prefix as it felt redundant. Please let me know if there was a reason for using it.

Reply:
Ah! I was worried about this as well. Before this PR, the API paths were being serviced directly from `LMS_HOST/api/v1/learning-paths/` using the LearningPaths model as a reference in the URL. When working on the PR, I noticed from swagger docs that plugins usually follow `LMS_HOST/api/<plugin-name>/v1/..` as the pattern. So I added that to conform to the norms. This ended up with `/api/learning_paths/v1/learning-paths/..`.